Docker.PowerShell.sln

```diff
@@ -16,6 +14,14 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Docker.DotNet", "src\Docker
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{FD49E869-E8F2-4E40-837D-E0329F087D4A}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Docker.DotNet.X509", "src\Docker.DotNet\Docker.DotNet.X509\Docker.DotNet.X509.xproj", "{3EB8C108-EAD7-4A58-AACF-33D9149F4708}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Tar", "src\Tar\Tar.xproj", "{FAC0B799-8BF2-4FCE-8867-D717867403A1}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{42C1BB74-7512-4B4B-AF57-F1448CD7A337}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "src", "test\src\src.xproj", "{B7044204-872F-41F0-A98A-232A43572089}"
```

Comment:
In my last review I asked about the name of this. Do we really want to call it src?